packages/vm/src/edge-vm.ts

```diff
@@ -72,12 +76,52 @@ export class EdgeVM<T extends EdgeContext = EdgeContext> extends VM<T> {
 
     this.evaluate<void>(getDefineEventListenersCode())
     this.dispatchFetch = this.evaluate<DispatchFetch>(getDispatchFetchCode())
+    for (const item of [
```

Comment:
Do you mind adding a brief comment why we need to patch these specific ones? 🙏

Reply:
<picture data-single-emoji=":done:" title=":done:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/done/556da3823bdfab37.png" alt=":done:" align="absmiddle"></picture>